gulpfile.js

```diff
@@ -15,16 +17,38 @@ var lintPaths = [
     'gulpfile.js'
 ];
 
-gulp.task('test', function(done) {
+gulp.task('serve:backend', function () {
+    process.env.PYTHONUNBUFFERED = 1;
+    process.env.PYTHONDONTWRITEBITECODE = 1;
+    spawn('python', ['manage.py', 'runserver'], {
```

Comment:
@jgmize can you think of a reason to **not** have the default be `"0.0.0.0:$PORT"`, with a default value for the `PORT` env var of `8000`? That's what I'm leaning toward, but if we need to default to 127.0.0.1 then I can work that out as well.